tabular/ranking/data_urls.csv

```diff
@@ -0,0 +1,3 @@
+https://www.dropbox.com/s/h3aazp8tlzweiks/eval.csv?dl=0,eval.csv
+https://www.dropbox.com/s/w26urcheglwahqz/ref.csv?dl=0 -O /content/ref.csv,ref.csv

```

Comment:
you don't need the -O, the script does the wget with the parameters by parsing this csv file 

also all links should have ?dl=1 at the end not ?dl=0